src/Api/Controllers/UsersController.Picture.cs

```diff
@@ -151,46 +107,25 @@ private static bool IsImage(IFormFile file)
         // Delete api/users/:id/picture/
 
         /// <summary>
-        /// Delete picture for the user {id}
+        /// Delete picture for the current user
         /// </summary>        
-        /// <param name="id">The Id of the user</param>                                 
+        /// <param name="identy">The identy of the user to be delete (For admins)</param>                                 
         /// <response code="200">Request is accepted</response>        
         /// <response code="400">Request incorrect</response>        
         /// <response code="403">User not allowed to delete picture</response>                
         /// <response code="401">User is denied</response>
-        [HttpDelete("{id}/picture/")]
```

Comment:
No:
        public IActionResult Delete([FromQuery] string identity)
        {
            if (identity == null && !_userPermissions.IsAllowedToAdminister(User.Identity.GetUserIdentity()))
                return Forbidden();

"[FromQuery] string identity" is an optinal Parameter, and only Admins can specify the identity.